pymbar/mbar_solvers.py

```diff
@@ -3,18 +3,49 @@
 import numpy as np
 import math
 import scipy.optimize
-from pymbar.utils import ensure_type, logsumexp, check_w_normalized
+from pymbar.utils import ensure_type, logsumexp, check_w_normalized, ParameterError
 import warnings
 
 logger = logging.getLogger(__name__)
 
 # Below are the recommended default protocols (ordered sequence of minimization algorithms / NLE solvers) for solving the MBAR equations.
 # Note: we use tuples instead of lists to avoid accidental mutability.
-# DEFAULT_SUBSAMPLING_PROTOCOL = (dict(method="L-BFGS-B"), )  # First use BFGS on subsampled data.
-# DEFAULT_SOLVER_PROTOCOL = (dict(method="hybr"), )  # Then do fmin hybrid on full dataset.
-# Use Adpative solver as first attempt
-DEFAULT_SOLVER_METHOD = "adaptive"
-DEFAULT_SOLVER_PROTOCOL = (dict(method=DEFAULT_SOLVER_METHOD),)
+JAX_SOLVER_PROTOCOL = (
+    dict(method="BFGS", continuation=True),
+    dict(method="adaptive", options=dict(min_sc_iter=0)),
+)
+
+DEFAULT_SOLVER_PROTOCOL = (
+    dict(method="hybr", continuation=True),
+    dict(method="adaptive", options=dict(min_sc_iter=0)),
+)
+
+ROBUST_SOLVER_PROTOCOL = (
```

Comment:
Shouldn't "robust" be default, with faster methods available if desired?

Reply:
Good question.  `default` is essentially what was done before. I can play around with this a little.